Example/PaymentSheet Example/PaymentSheet Example/CustomerSessionPlaygroundView.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  CustomerSessionPlaygroundView.swift
+//  PaymentSheet Example
+//
+
+import StripePaymentSheet
+
+import SwiftUI
+
+@available(iOS 14.0, *)
+struct CustomerSessionPlaygroundView: View {
+    @State var viewModel: PaymentSheetTestPlaygroundSettings
+    var doneAction: ((PaymentSheetTestPlaygroundSettings) -> Void) = { _ in }
+
+    init(viewModel: PaymentSheetTestPlaygroundSettings, doneAction: @escaping ((PaymentSheetTestPlaygroundSettings) -> Void)) {
+        _viewModel = State<PaymentSheetTestPlaygroundSettings>.init(initialValue: viewModel)
+        self.doneAction = doneAction
+    }
```

Comment:
Do we not get this initializer for free?

Reply:
ah! you're right. removing!